Android/Java/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.java

```diff
@@ -202,38 +193,52 @@ public void setBannerViewVisibility(int visibility) {
     }
     //endregion
 
-    //region Rewarded Video Methods
-    public void showRewardedVideoButtonTapped(View view) {
-        // It is advised to make sure there is available ad before attempting to show an ad
-        if (IronSource.isRewardedVideoAvailable()) {
-            // This will present the Rewarded Video.
+    //region Rewarded  Methods
```

Comment:
I think it's better to move this region to right after the interstitial section, because they are pretty similar ads than banners